source/stylesheets/base/elements/typography.styl

```diff
@@ -64,6 +70,28 @@ a:active {
     text-decoration: underline;
 }
 
+@media print {
+    a {
+        @extends $print-plain;
+
+        &[href^='http'] { // print links on page for off site links
+            text-decoration: underline !important;
+            word-wrap: break-word;
```

Comment:
We should reset the text-transform on printed links. While they look nice on the page, e.g. 

HTTPS://WWW.FACEBOOK.COM/EVENTS/1726468307566424/

...when used verbatim, this URL becomes...

https://www.facebook.com/EVENTS/1726468307566424/

... which is broken.